dropwizard-request-logging/src/main/java/io/dropwizard/request/logging/filter/UrlFilterFactory.java

```diff
@@ -0,0 +1,43 @@
+package io.dropwizard.request.logging.filter;
+
+import ch.qos.logback.access.spi.IAccessEvent;
+import ch.qos.logback.core.filter.Filter;
+import ch.qos.logback.core.spi.FilterReply;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import io.dropwizard.logging.filter.FilterFactory;
+
+import java.util.Collections;
+import java.util.Set;
+
+import javax.validation.constraints.NotNull;
+
+@JsonTypeName("url-filter-factory")
```

Comment:
For consistency with the server factories this can be shortened to uri.

https://github.com/dropwizard/dropwizard/blob/c05a7aa6cff717683ccf9aa43130c5eb3a7a1c45/dropwizard-core/src/main/java/io/dropwizard/server/DefaultServerFactory.java#L69-L70

https://github.com/dropwizard/dropwizard/blob/c05a7aa6cff717683ccf9aa43130c5eb3a7a1c45/dropwizard-core/src/main/java/io/dropwizard/server/SimpleServerFactory.java#L56-L57

Reply:
I see in the docs an example request logging filter has the suffix `-filter-factory`, so I'm a little unsure if there is a convention for log filter factories, or if I should just follow other factory naming conventions.

Reply:
I'll switch to `uri` as the type, unless I hear there is a different naming convention though.